pxf/build.gradle

```diff
@@ -218,7 +218,7 @@ project('pxf-service') {
 
 project('pxf-hdfs') {
     dependencies {
-        compile(project(':pxf-service')) //Yikes, HdfsAnalyzer is directly accessing the bridge
+        compile(project(':pxf-service'))
```

Comment:
Can't we replace the dependency with a dependency on pxf-api (like pxf-hbase does)?